### ISSUE TITLE
shimming a jquery plugin

### ISSUE THREAD

**Issue by newtonianb** [webpack/webpack#1282]
I'm trying to shim this plugin called jquery-stickit https://github.com/garand/sticky

I got to this point using the imports-loader to provide the jQuery variable inside the script.

```
$ = require('jquery')
require("imports?jQuery=jquery!./jquery-sticky/jquery.sticky");
$('.test').stickit();
```

I'm still not sure how I can use this though I get 
``Uncaught TypeError: $(...).sticky is not a function``

**Comment by SimenB** [comment]
Have you tried http://webpack.github.io/docs/shimming-modules.html#plugin-provideplugin?

**Comment by sokra** [comment]
You wrote `$('.test').stickit();` instead of `$('.test').sticky();`

**Comment by newtonianb** [comment]
woops good observation thanks! I was testing both libraries they both exist but seems my code was out of date, let me try again and verify.

**Comment by newtonianb** [comment]
@SimenB thanks for the info but I don't want to make jQuery global

**Comment by bebraw** [comment]
@newtonianb If this is still relevant, please open a question at Stack Overflow. There are likely related answers already btw.
